src/main/kotlin/fi/metatavu/timebank/api/forecast/ForecastService.kt

```diff
@@ -43,14 +44,27 @@ class ForecastService {
             val response = client.newCall(request).execute()
             when (response.code()) {
                 200 -> response.body()?.string()
-                else -> throw Error("Couldn't reach Forecast API.")
+                else -> throw Error("Couldn't reach Severa API.")
             }
         } catch (e: Error) {
             logger.error("Error when executing get request: ${e.localizedMessage}")
             throw Error(e.localizedMessage)
         }
     }
 
+    /**
+     * Finds ForecastPerson from Forecast by id
+     *
+     * @param personId Int
+     * @return ForecastPerson?
+     */
+    fun findPerson(personId: Int): ForecastPerson {
```

Comment:
Since this whole `Persons` resource is being deprecated in this PR and given the fact that our integration with this service ends in 7 days (at the time of writing this) I'd say we don't want to do **any** work in this class.